### PR TITLE
Massinvitations

### PR DESCRIPTION
**Pull request by aomran**
For https://github.com/aomra015/curry/issues/2

### CODE REVIEW

File: app/controllers/invitations_controller.rb [hunk 1]
```diff
@@ -11,16 +11,27 @@ def new
   end
 
   def create
-    @invitation = Invitation.new(email: params[:invitation][:email])
-    @invitation.classroom = @classroom
+    emails = params[:invitation_emails].split(/,|\n/)
+    emails.each { |email| email.strip! }
```
Comment (pjfranzini) [inline app/controllers/invitations_controller.rb]: this line could be included in the similar loop below

File: app/controllers/invitations_controller.rb [hunk 2]
```diff
@@ -11,16 +11,27 @@ def new
...
+    invitations = []
 
-    if @invitation.save
-      InvitationMailer.invite(@invitation).deliver
-      redirect_to classroom_tracks_path(@classroom), notice: 'Invitation Sent'
-    else
-      render :new
+    emails.each do |email|
+      invitation = Invitation.new(email: email)
+      invitation.classroom = @classroom
```
Comment (pjfranzini) [inline app/controllers/invitations_controller.rb:19]: Could be more efficient to check invalidity of invitation already here

File: app/controllers/invitations_controller.rb [hunk 3]
```diff
@@ -77,6 +88,16 @@ def add_student
 
   end
 
+  def destroy
+    invitation = @classroom.invitations.find(params[:id])
+    if invitation.destroy
+      redirect_to new_classroom_invitation_url(@classroom), notice: 'Invitation Removed'
+    else
+      @invitations = @classroom.invitations
```
Comment (pjfranzini) [inline app/controllers/invitations_controller.rb]: When would this branch ever happen?

File: app/views/invitations/new.html.erb [hunk 4]
```diff
@@ -1,15 +1,14 @@
 <div class="grid1">
 <h1>Invite Students</h1>
-<%= form_for [@classroom, @invitation] do |f| %>
-  <%= f.error_messages %>
+<%= form_tag classroom_invitations_path(@classroom) do |f| %>
```
Comment (pjfranzini) [inline app/views/invitations/new.html.erb]: no |f| here?

**Comment by pjfranzini** [comment]
Since the "Invite students" button is also the way to access the table where one sees who one has invited, the status of the invite, and the possibility to remove an invite, should the button be labeled something more fully informative, eg "Manage invitations"?

Edit: Loaded up branch and tested out functionality of multiple invitations, of validation of same, and of a student no longer seeing classroom if their invitation is deleted; all seems to work properly.

**Comment by aomran** [comment]
I've gone ahead and made the fixes.

**Comment by pjfranzini** [comment]
Looks good.

**Pull request merged**